src/transformers/models/roberta/convert_roberta_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -35,8 +35,8 @@
 from transformers.utils import logging
 
 
-if version.parse(fairseq.__version__) < version.parse("0.9.0"):
-    raise Exception("requires fairseq >= 0.9.0")
+if version.parse(fairseq.__version__) < version.parse("1.0.0a"):
+    raise Exception("requires fairseq >= 1.0.0a")
```

Comment:
it seems it is miss typo for 0.10.0a?

Reply:
Hi @Soonhwan-Kwon, latest `fairseq` master is needed and it is currently tagged as `1.0.0a`, see [here](https://github.com/pytorch/fairseq/blob/master/fairseq/version.txt)

Reply:
Thank you for the confirmation, latest pip version is 0.10.2, so I confused. It needs to be build in source to get 1.0.0a.